src/CabanaPD_Types.hpp

```diff
@@ -14,19 +14,27 @@
 
 namespace CabanaPD
 {
+// Mechanics types.
 struct Elastic
 {
 };
 struct Fracture
 {
 };
+
+// Thermal types.
 struct TemperatureDependent
 {
 };
 struct TemperatureIndependent
```

Comment:
Perhaps it would be better to change the order:
TemperatureIndependent
TemperatureDependent
DynamicTemperature

Reply:
Done